docs/content/en/locale-fallback.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Locale fallback
+description: "How a fallback gets selected when a translation is missing"
+position: 11
+category: Guide
+---
+
+## Locale fallback
```

Comment:
This is redundant and results in two identical headers on this page.

Reply:
Job's done